### ISSUE TITLE
Does javascript needs to be compiled?

### ISSUE THREAD

**Issue by bofeiw** [shamsmosowi/git-bot#26]

**Comment by shamsmosowi** [comment]
No, JavaScript does not need to be compiled. JavaScript is an interpreted language, meaning that it is read directly by the browser and does not need to be compiled into a different language before being run. This makes it faster and easier to use than other programming languages that require compilation. 

However, JavaScript can be optimized or minified to reduce the size of the code and make it load faster. This involves removing unnecessary characters and formatting the code in a way that makes it easier to read. Additionally, some JavaScript frameworks, such as TypeScript, compile JavaScript code into other languages, such as Java or C#. 

I hope this helps!
